packages/sdk/test/services/credential-service.test.ts

```diff
@@ -862,7 +1130,7 @@ describe('CredentialService', () => {
               },
             ],
             classification_nodes: [],
-            label: 'urn:uuid:66cfde0a-bac9-4ed3-8cee-10e4a417dcc9',
```

Comment:
why label has changed to the value without `urn:uuid`?

Reply:
@linasi 
Initially, I removed the `formatIdToItemName` function, thinking the client would provide the correct ID. However, upon reflection, I've decided to bring back the function to handle ID formatting within the SDK.